### ISSUE TITLE
Reduce space between bar charts on content type landing pages

### ISSUE THREAD

**Issue by juliandsmith** [AASHE/hub#386]
On content-type landing pages with multiple bar charts (e.g., case studies and publications), there seems to be an extraordinary amount of space between the two charts, as shown below. Can we reduce this to a more reasonable amount of space (maybe one or two breaks?).

![image](https://cloud.githubusercontent.com/assets/15839046/18231214/9cc32b62-7280-11e6-95ec-423402cf1860.png)

**Comment by baronvonvaderham** [comment]
Finally figured this out. Google's documentation is awful......

This is specified using the ChartArea attribute in the Options variable. The chart height is set correctly using "# of rows  * 25pixels + 80pixels". That seems to be the sweet spot to get bar widths that exactly match the label height.

But the ChartArea apparently only controls the BAR AREA, not the ENTIRE chart, which any sane person would take to mean the axes, labels, legend, etc. Specifying "left: 25%" for the labels is fine, because the width of that div is fixed, and then it scales nicely on mobile. But I didn't fully grasp how this was interacting with the labels, and because of that fixed width, it didn't give me any clues.

I was using that same format to specify the top and bottom margins. Since that percentage is relative to the CHART size, on the vertical axis that's variable, and dependent on the number of bars we have. 

That's why everything looked peachy locally and on dev, where we truncate the DB to only have 5-6 items per chart! The chart height is then 250pixels or so, and the margin of 5-10% is reasonable. Then we deployed that into production, and now the height is 700+ pixels, so that same percentage is now a HUGE margin....doubled when we have two charts present.

Now that I know what the hell those attributes REALLY mean, no thanks to google's documentation, the fix was easy enough. Our font size for the title and x-axis labels is fixed, so we'll always need the same buffer. I just set those margin attributes equal to absolute pixel counts that give that text JUST enough room to be displayed.
